docs/developer-docs/latest/development/custom-fields.md

```diff
@@ -198,6 +223,47 @@ Each object in the `items` array can contain the following parameters:
 | `intlLabel`     | Translation for the label of the input                             | [`IntlObject`](https://formatjs.io/docs/react-intl/) |
 | `type`          | Type of the input (e.g., `select`, `checkbox`)                     | `String`                                             |
 
+::: details "type" accepts the following inputs:
```

Comment:
We use the details block for code examples on this page, so for consistency, I wouldn't use it here. Since we're describing the different values accepted for `type`, I'd suggest removing this block and expanding the table above with a list.

You can use something like the following for line 224 :)
```
| `type` | Type of the input.<br />Accepts the following values:  <ul><li>'text',</li><li>'checkbox',</li><li>'checkbox-with-number-field',</li><li>'boolean-radio-group',</li><li>'select-number',</li><li>'radio-group',</li><li>'select',</li><li>'select-default-boolean',</li></li>'select-date',</li><li>'textarea-enum',</li></ul>	 | `String`
```